plugins/rtmp-services/data/services.json

```diff
@@ -654,6 +654,81 @@
                 "keyint": 2
             }
         },
+        {
+            "name": "Livepush",
+            "common": false,
+            "stream_key_link": "https://docs.livepush.io/en/articles/5065323-how-to-stream-live-from-obs-to-livepush",
```

Comment:
Since this does not link to a page showing the stream key, this should instead use the `more_info_link` field.

Reply:
Done. Thanks for the tip!